### DISCUSSION TITLE
Enable Yubikey for 2FA

### DISCUSSION THREAD

**Discussion by shanebrewer** [guerzon/vaultwarden#65] [Q&A] [answered]
I've been trying to enable Yubikeys which requires the YUBICO_CLIENT_ID and YUBICO_SECRET_KEY environment variables to be set. However, I've been unsuccessful in getting them set up by adding them to the values.yaml file. I've tried:

Attempt #1
env:
  YUBICO_CLIENT_ID: XXXX
  YUBICO_SECRET_KEY: YYYY
  
Attempt #2
YUBICO_CLIENT_ID:
  value: XXXX
YUBICO_SECRET_KEY:
  value: YYYY

But neither works. Is there a format that works, or is this just not supported by this Helm chart? 

Thanks!

**Comment by guerzon** [comment, marked as answer]
Hey @shanebrewer,

I don't know where you're adding those values, but I couldn't imagine how they would work.

In #66, I have just added support for [Yubikeys](https://github.com/dani-garcia/vaultwarden/wiki/Enabling-Yubikey-OTP-authentication).

```bash
helm repo update
```

Just add the following to your custom values yaml and upgrade:

```yaml
yubico:
  clientId: "abc"
  secretKey: "xyz"
```

br,
Lester
